src/mono/sample/iOS/Program.csproj

```diff
@@ -97,6 +98,10 @@
         <Output TaskParameter="XcodeProjectPath" PropertyName="XcodeProjectPath" />
     </AppleAppBuilderTask>
 
+    <Exec Condition="'$(StripDebugSymbols)' == 'true'"
+          Command="dsymutil $(AppBundlePath)/$(AppName)
```

Comment:
Where will the output of `dsymutil` be generated? Can that cause issues?

Reply:
As output file is not provided, it will be generated in `$(AppBundlePath)` directory with `$(AppName)` name and `.dSYM` extension.

Reply:
I might be wrong, but I think this way `dSYM` would become part of the `.app` bundle. It would be good to check this.

Reply:
Correct, it is part of `.app` bundle. Here is an example:
```
publish/app/HelloiOS/Release-iphoneos/HelloiOS.app/HelloiOS
publish/app/HelloiOS/Release-iphoneos/HelloiOS.app/HelloiOS.dSYM
```
Should it be excluded from the bundle?

Reply:
Yes. As you suggested, specifying the output file would resolve this.